examples/l3out/comprehensive/simplified/vrf.tf

```diff
@@ -0,0 +1,29 @@
+resource "aci_vrf" "vrf" {
+  tenant_dn = aci_tenant.tenant.id
+  name      = "vrf1"
+}
+
+resource "aci_vrf" "vrf2" {
+  tenant_dn = aci_tenant.tenant.id
+  name      = "vrf2"
+}
+
+resource "aci_vrf" "vrf3" {
+  tenant_dn = aci_tenant.tenant.id
+  name      = "vrf3"
+}
+
+resource "aci_vrf" "vrf4" {
+  tenant_dn = aci_tenant.tenant.id
+  name      = "vrf4"
+}
+
+resource "aci_vrf" "vrf5" {
+  tenant_dn = aci_tenant.tenant.id
+  name      = "vrf5"
+}
+
+resource "aci_vrf" "vrf6" {
+  tenant_dn = aci_tenant.tenant.id
+  name      = "vrf6"
+}
```

Comment:
Does this not cause parallelism issue?

Reply:
All the resources have the same reference to the parent object. There is no reason for a parallelism issue here.